synapse/push/httppusher.py

```diff
@@ -101,6 +104,7 @@ def __init__(self, hs: "HomeServer", pusher_config: PusherConfig):
         self._storage_controllers = self.hs.get_storage_controllers()
         self.app_display_name = pusher_config.app_display_name
         self.device_display_name = pusher_config.device_display_name
+        self.device_id = pusher_config.device_id
```

Comment:
I think this is unused?

Reply:
It is but in the `ModuleApi`, not inside the `HttpPusher`.